app/renderer/components/preferences/payment/bitcoinDashboard.js

```diff
@@ -341,7 +341,7 @@ class BitcoinDashboard extends ImmutableComponent {
                         {/* TODO: refactor button.js */}
                         <Button className={cx({
                           primaryButton: true,
-                          [css(globalStyles.spacing.dialogInsideMargin)]: true
+                          [css(styles.bitcoinAddressButton)]: true
```

Comment:
@NejcZdovc @bsclifton could you both left a comment on https://github.com/brave/browser-laptop/issues/8340 regarding BEM? We should standardize our Aphrodite stuff as soon as we can to avoid future rework. @luixxiul left some great insights there.